backend/controller/internal/dal/events.go

```diff
@@ -110,7 +110,7 @@ func FilterTypes(types ...sql.EventType) EventFilter {
 
 type eventRow struct {
 	DeploymentKey sqltypes.Key
-	RequestKey    sqltypes.Key
```

Comment:
Don't use pointers, use `types.Option` instead. If you use a pointer it's ambiguous as to whether it's "this is optional" or "this is just a pointer to a struct".